src/test/java/com/algorand/algosdk/cucumber/shared/TransactionSteps.java

```diff
@@ -99,9 +108,9 @@ public void buildApplicationTransactions(String operation, Long applicationId, S
         if (StringUtils.isNotEmpty(appAccounts)) {
             builder.accounts(convertAccounts(appAccounts));
         }
-        if (fee != 0) {
-            builder.flatFee(fee);
-        }
+
+        builder.flatFee(fee);
```

Comment:
Not a huge deal, but I would move this up to line 93 so the property setters and conditionals are each grouped together.